source/jobs/stopping-jobs.txt

```diff
@@ -1,5 +1,64 @@
+.. _rm-stop-jobs:
+
 ===============
 Stop a Sync Job
 ===============
 
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can stop a sync job from the :guilabel:`Job Overview` pane on the 
+:guilabel:`Data Migration` tab.
+
+About this Task
+---------------
+
+The effects of stopping a sync job on your destination database depend on 
+the type and stage of the sync job:
+
+- If you stop a snapshot sync job during the snapshot stage, any 
+  partially migrated data remains in the MongoDB database.
+- If you stop a continuous sync job during the CDC stage, the process 
+  ends gracefully, and Relational Migrator cleans up any temporary data.
+
+Once initiated, the process of canceling or completing a sync job may take up to one minute.
+
+Steps
+-----
+
+To stop your sync job, follow the instructions below based on the current stage of execution:
+
```

Comment:
(praise)
I really like the way you've set up this page. This is a nice use of space and looks to be a very good user experience!